.github/actions/publish_module/action.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Publish Module'
+description: 'Publishes module'
+inputs:
+  module:
+    description: 'Module'
+    required: true
+    default: ''
+  sonatype_usr:
+    description: 'Sonatype user'
+    required: true
+    default: ''
+  sonatype_pwd:
+    description: 'Sonatype password'
+    required: true
+    default: ''
+  signing_key_id:
+    description: 'Signing key id'
+    required: true
+    default: ''
+  signing_key_pwd:
+    description: 'Signing key password'
+    required: true
+    default: ''
+  signing_key_file:
+    description: 'Signing key file'
+    required: true
+    default: ''
+runs:
+  using: "composite"
+  steps:
+    - run: ./gradlew --stacktrace clean :${{ inputs.module }}:publishToSonatype
+      shell: bash
+      env:
+        SONATYPE_USERNAME: ${{ inputs.sonatype_usr }}
+        SONATYPE_PASSWORD: ${{ inputs.sonatype_pwd }}
+        SIGNING_KEY_ID: ${{ inputs.signing_key_id }}
+        SIGNING_KEY_PASSWORD: ${{ inputs.signing_key_pwd }}
+        SIGNING_KEY_FILE: ${{ inputs.signing_key_file }}
+    - run: ./gradlew --stacktrace closeAndReleaseRepository
```

Comment:
This step hasn't been tested - we tested up to `publishToSonatype` and manually dropped the repos from Sonatype without releasing. The `closeAndReleaseRepository` step should actually release the SDK, so we did not want to do that in testing.